spark/v3.4/spark/src/test/java/org/apache/iceberg/spark/actions/TestCreateActions.java

```diff
@@ -972,33 +974,38 @@ private void assertSnapshotFileCount(SnapshotTable snapshotTable, String source,
     long expectedFiles = expectedFilesCount(source);
     SnapshotTable.Result snapshotTableResult = snapshotTable.execute();
     validateTables(source, dest);
-    Assert.assertEquals(
-        "Expected number of imported snapshot files",
-        expectedFiles,
-        snapshotTableResult.importedDataFilesCount());
+    assertThat(snapshotTableResult.importedDataFilesCount())
+        .as("Expected number of imported snapshot files")
+        .isEqualTo(expectedFiles);
   }
 
   private void validateTables(String source, String dest)
       throws NoSuchTableException, ParseException {
     List<Row> expected = spark.table(source).collectAsList();
     SparkTable destTable = loadTable(dest);
-    Assert.assertEquals(
-        "Provider should be iceberg",
-        "iceberg",
-        destTable.properties().get(TableCatalog.PROP_PROVIDER));
+    assertThat(destTable.properties().get(TableCatalog.PROP_PROVIDER))
```

Comment:
it would be better to use `.containsEntry` here

Reply:
Thanks for checking this! Fix this part.